tests/UI/campaigns/regression/checkout/100PercentDiscount_FO.js

```diff
@@ -0,0 +1,417 @@
+ * not be disabled
+ *
+ * login to BO
+ * add new discount (-100% free shipping)
+ * Change terms and conditions setting
+ */
+describe('Create 100% discount with free shipping discount code', async () => {
+  // before and after functions
+  before(async function () {
+    browserContext = await helper.createBrowserContext(this.browser);
+    page = await helper.newTab(browserContext);
+  });
+
+  after(async () => {
+    await helper.closeBrowserContext(browserContext);
+  });
+
+  it('should login in BO', async function () {
+    await loginCommon.loginBO(this, page);
+  });
+
+  /**
+   * Setup, create new 100% discount code with free shipping
+   */
+  describe('SETUP', async () => {
+    it('should go to cart rule page', async function () {
+      await testContext.addContextItem(
+        this,
+        'testIdentifier',
+        'goToCartRulesPageToCreate',
+        baseContext,
+      );
+
+      await dashboardPage.goToSubMenu(
+        page,
+        dashboardPage.catalogParentLink,
+        dashboardPage.discountsLink,
+      );
+
+      const pageTitle = await cartRulesPage.getPageTitle(page);
+      await expect(pageTitle).to.contains(cartRulesPage.pageTitle);
+    });
+
+    describe('Create a percentage cart rule', async () => {
+      it('should go to new cart rule page', async function () {
+        await testContext.addContextItem(
+          this,
+          'testIdentifier',
+          'goToNewCartRulePage1',
+          baseContext,
+        );
+
+        await cartRulesPage.goToAddNewCartRulesPage(page);
+        const pageTitle = await addCartRulePage.getPageTitle(page);
+        await expect(pageTitle).to.contains(addCartRulePage.pageTitle);
+      });
+
+      it('should create new cart rule', async function () {
+        await testContext.addContextItem(
+          this,
+          'testIdentifier',
+          'createPercentCartRule',
+          baseContext,
+        );
+
+        const validationMessage = await addCartRulePage.createEditCartRules(
+          page,
+          percentCartRule,
+        );
+
+        await expect(validationMessage).to.contains(
+          addCartRulePage.successfulCreationMessage,
+        );
+      });
+    });
+
+    describe('Change terms and conditions setting no', async () => {
+      it("should go to 'Shop Parameters > Order Settings' page", async function () {
+        await testContext.addContextItem(
+          this,
+          'testIdentifier',
+          'goToOrderSettingsPage',
+          baseContext,
+        );
+
+        await dashboardPage.goToSubMenu(
+          page,
+          dashboardPage.shopParametersParentLink,
+          dashboardPage.orderSettingsLink,
+        );
+
+        const pageTitle = await orderSettingsPage.getPageTitle(page);
+        await expect(pageTitle).to.contains(orderSettingsPage.pageTitle);
+      });
+
+      it('Should change the terms and conditions back to disabled', async () => {
```

Comment:
```suggestion
      it('should change the terms and conditions back to disabled', async function () {
```

This should fix the error on GA.